ui/src/transaction.ts

```diff
@@ -323,12 +376,54 @@ interface Payer {
   imageUrl?: string;
 }
 
-export const formatPayer = (identity: Identity): Payer =>
-  identity && {
-    name: identity.metadata.handle,
+export const payerFromIdentity = (identity: Identity): Payer => {
+  return {
+    name: identity.registered ?? identity.metadata.handle,
     type: PayerType.User,
     avatarFallback: identity.avatarFallback,
   };
+};
+
+const payerFromOrg = (org: Org): Payer => {
+  return {
+    name: org.id,
+    type: PayerType.Org,
+    avatarFallback: org.avatarFallback,
+  };
+};
+
+// Get the payer of a transaction.
+// Note: It now look the payer up based on the local session, whereas
+// in the future we want to look it up on the network.
+export const getPayer = (msg: Message, session: Session): Payer | undefined => {
```

Comment:
A comment for peers and reviewers: This function was motivated by a bug found where we were _always_ passing a payer value that `Transaction.svelte` expects from the `session.identity`. I.e, we were saying the payer was always the identity stored on the machine. 

For screens _creating_ a transaction, such as `ProjectRegistration`, `OrgRegistration`, etc, we implicitly know who pays for it (in the registry). However, in `TransactionDetails` we have no context, only having an transaction `id` to work with, with we then use to fetch said transaction. Here, we need to figure who paid for that transaction, and we do that here. 

So since we _must_ use `getPayer` in `TransactionDetails`, I figured best to use it everywhere to have one source of truth and logic. That's the trade-off behind it.